samples/drm/app/showcase_libraries.json

```diff
@@ -0,0 +1,24 @@
+{
+    "items": [
+        {
+            "name": "jquery",
+            "link": "http://jquery.com/"
+        },
+        {
+            "name": "angular",
+            "link": "http://angularjs.org/"
+        },
+        {
+            "name": "bootstrap",
+            "link": "http://getbootstrap.com/"
+        },
+        {
+            "name": "flot",
+            "link": "http://www.flotcharts.org/"
+        },
+        {
+            "name": "angular treeview",
+            "link": "https://github.com/eu81273/angular.treeview"
+        }
+    ]
+}
```

Comment:
I was thinking of this new demo page as being a pretty flashy thing that we might want to show at conferences and other venues, so I didn't want to lose our UI components that recognize libraries we use and companies that have contributed.  Maybe there is a common place we should put this stuff that any of our demo apps can access?